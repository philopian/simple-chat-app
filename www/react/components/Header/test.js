import React from "react";
import Enzyme, { configure, shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import "raf/polyfill";
import Adapter from "enzyme-adapter-react-16";
import Component from "./index.js";

Enzyme.configure({ adapter: new Adapter() });

describe("Header (Snapshot)", () => {
  it("Header renders without crashing", () => {
    const component = renderer.create(<Component />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});

describe("Header Component - ", () => {
  it("Make sure the props are being passing thru to the component", () => {
    // Render the component
    const component = shallow(<Component title="some title here!" />);

    // Make sure that the props.message is being printed
    expect(component.find("p").text()).toEqual("some title here!");
  });
});
