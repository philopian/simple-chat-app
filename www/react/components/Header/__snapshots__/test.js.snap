// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header (Snapshot) Header renders without crashing 1`] = `
<div
  className="sc-bdVaJa eyqDlD"
>
  <p />
</div>
`;
